torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -125,48 +170,157 @@ void insertMkldnnPrePackedOps(script::Module& module) {
   }
 }
 
-void FuseReluWithPackedOps(std::shared_ptr<Graph>& graph) {
+template <typename T>
+void RewriteEltwiseGraph(
```

Comment:
Why is it a template? Isnt `T` always `mkldnn::PostOp`?

Reply:
the template is not needed, removed.